pkg/adaptor/mongodb/mongodb.go

```diff
@@ -359,35 +359,36 @@ func (m *MongoDB) catData() error {
 		} else if match := m.collectionMatch.MatchString(collection); !match {
 			continue
 		}
-		query := bson.M{}
-		// lastID := ""
+		canReissueQuery := m.requeryable(collection)
```

Comment:
Hmm, this should be okay to cache. Hopefully people aren't messing with their indices while transporter is running :scary: